app/src/main/java/com/codepath/travel/activities/CreateStoryActivity.java

```diff
@@ -147,6 +171,38 @@ private void setUpRecyclerView() {
         mItemTouchHelper.attachToRecyclerView(rvStoryPlaces);
     }
 
+    private void addSelectedPlaceInTrip(Place place, String photoReference) {
+        StoryPlace storyPlace = new StoryPlace(mNewTrip, place);
+        storyPlace.setThumbnail(photoReference);
```

Comment:
Sorry, that was my bad. I should have made the view update. will try to add that tonight or tomorrow.